presto-main/src/main/java/com/facebook/presto/sql/planner/plan/TableScanNode.java

```diff
@@ -40,6 +41,7 @@
 {
     private final TableHandle table;
     private final List<Symbol> outputSymbols;
+    private final List<VariableReferenceExpression> outputVariables;
     private final Map<Symbol, ColumnHandle> assignments; // symbol -> column
```

Comment:
This should be Symbol as well?

Reply:
All Symbol in all PlanNode should be variables. They can't all happen in one diff.